src/engine/strat_engine/cmd.rs

```diff
@@ -14,6 +14,9 @@
 // the existence of the file is checked before the command is invoked, and
 // an explicit error is returned if the executable can not be found.
 
+use libc::c_uint;
+use libcryptsetup_rs::SafeMemHandle;
+use serde_json::Value;
```

Comment:
We generally want the imports to be separated into the following categories:

`std`

external crates (like libc, serde_json)

crates that are external that we develop (like devicemapper)

`crate`

all separated by a newline so they're ordered appropriately within those categories.